### PR TITLE
feat: New Microsoft Teams alert channel

### PR DESCRIPTION
**Pull request by dmurray-lacework**
Microsoft Teams alert channel

Signed-off-by: Darren Murray <darren.murray@lacework.net>

### CODE REVIEW

**Review by scottford-lw — APPROVED** [review]
👍

File: cli/cmd/integration_microsoft_teams.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+func createMicrosoftTeamsChannelIntegration() error {
+	questions := []*survey.Question{
+		{
+			Name:     "name",
+			Prompt:   &survey.Input{Message: "Name: "},
+			Validate: survey.Required,
+		},
+		{
+			Name:     "teams url",
+			Prompt:   &survey.Input{Message: "Teams URL: "},
```
Comment (afiune) [inline cli/cmd/integration_microsoft_teams.go:36]: should we add the word "Webhook" somehow here? https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook

A suggestion would be `Webhook URL` or `MS Teams Webhook URL`. 🤔

File: cli/cmd/integration_microsoft_teams.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+func createMicrosoftTeamsChannelIntegration() error {
+	questions := []*survey.Question{
+		{
+			Name:     "name",
+			Prompt:   &survey.Input{Message: "Name: "},
+			Validate: survey.Required,
+		},
+		{
+			Name:     "teams url",
```
Comment (afiune) [inline cli/cmd/integration_microsoft_teams.go:35]: I think this should be only `url` instead so that it maps the answers, did you test this?

**Review by afiune — COMMENTED** [review]
A few comments/suggestions. Great work Darren!

tenor-23875480

File: cli/cmd/integration_microsoft_teams.go (hunk 2 above)
Comment (dmurray-lacework) [reply cli/cmd/integration_microsoft_teams.go]: Fixed!

File: cli/cmd/integration_microsoft_teams.go (hunk 1 above)
Comment (dmurray-lacework) [reply cli/cmd/integration_microsoft_teams.go]: Updated to webhook url

**Review by afiune — APPROVED** [review]
tenor-106165265

**Pull request merged**